src/App.js

```diff
@@ -1,27 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+// import './App.css';
```

Comment:
- [ ] Please remove this and any other commented code that you may have to keep your codebase clean.